src/overlays/actors/ovl_En_Bird/z_en_bird.c

```diff
@@ -47,81 +47,76 @@ void EnBird_Init(Actor* thisx, PlayState* play) {
     SkelAnime_Init(play, &this->skelAnime, &gBirdSkel, &gBirdFlyAnim, NULL, NULL, 0);
     ActorShape_Init(&this->actor.shape, 5500, ActorShadow_DrawCircle, 4);
     this->unk_194 = 0;
-    this->unk_198 = 0;
-    this->unk_1C0 = 0x9C4;
+    this->timer = 0;
+    this->rotYStep = 2500;
     this->actor.colChkInfo.mass = 0;
-    this->unk_1A8 = 1.5f;
-    this->unk_1AC = 0.5f;
-    this->unk_1A0 = 0.0f;
-    this->unk_1A4 = 0.0f;
-    this->unk_1B8 = 0.0f;
-    this->unk_1B0 = 40.0f;
+    this->speedXZTarget = 1.5f;
+    this->speedXZStep = 0.5f;
+    this->posYMag = 0.0f;
+    this->rotYMag = 0.0f;
+    this->posYPhaseStep = 0.0f;
+    this->flightDistance = 40.0f;
     this->unk_1BC = 70.0f;
-    func_809C1CAC(this, this->actor.params);
+    EnBird_SetupIdle(this, this->actor.params);
 }
 
 void EnBird_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void func_809C1CAC(EnBird* this, s16 params) {
+void EnBird_SetupIdle(EnBird* this, s16 params) {
     f32 frameCount = Animation_GetLastFrame(&gBirdFlyAnim);
-    f32 playbackSpeed = this->unk_19C ? 0.0f : 1.0f;
+    f32 playbackSpeed = this->aniSpeedMod ? 0.0f : 1.0f;
     AnimationHeader* anim = &gBirdFlyAnim;
```

Comment:
```suggestion
```
This seems to not be needed, and could just use `&gBirdFlyAnim` directly in the function call.